Components/Toast/toast.css

```diff
@@ -0,0 +1,103 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+:root {
+    --toast-bg-color: #333;
+    --toast-color: #fff;
+    --toast-icon-color: #fff;
+    --toast-button-color: #93c5fd;
+    /* focus color is seen when you use tab key to access buttons or links */
+    --toast-button-focus-outline-color: #fff;
+}
+
+.toast {
+    width: 30rem;
+    max-width: 95%;
+    padding: 0.8em 1.2em;
+    padding-right: 0.6em;
+    background-color: var(--toast-bg-color);
+    border-radius: 0.3em;
+    position: fixed;
+    bottom: 1.2rem;
+    right: 1.2rem;
+
+    /* flex - properties */
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1.5em;
+}
+
+.toast.toast-left {
+    right: none;
+    left: 1.2rem;
+}
+
+@media screen and (max-width: 1030px) {
```

Comment:
Another suggestion - to club media queries and add it towards the end of the sheet.